packages/manager/apps/pci-ai-tools/src/data/api/ai/notebook/notebook.api.ts

```diff
@@ -35,6 +35,15 @@ export const startNotebook = async ({
     .then((res) => res.data as ai.notebook.Notebook);
 };
 
+export const reStartNotebook = async ({
```

Comment:
Suggestion: since 'restart' is a word it's a little weird to have it written like 'reStart'
```suggestion
export const restartNotebook = async ({
```